src/utils/data_loader.rs

```diff
@@ -1,27 +1,22 @@
-use crate::utils::data_inserter::{InsertSmileyReport, InsertRestaurant, insert_restaurant, insert_smileys};
-use crate::utils::json_parser::{JsonRestaurant, JsonSmileyReport};
 use crate::database::new_pool;
+use crate::utils::data_inserter::{
+    insert_restaurant, insert_smileys, InsertRestaurant, InsertSmileyReport,
+};
+use crate::utils::json_parser::{JsonRestaurant, JsonSmileyReport};
 use std::fs::File;
 use std::io::BufReader;
 
-
 pub fn load_data(path: &String) {
     let file = File::open(path).expect("Can't open file from path");
     let reader = BufReader::new(file);
-    let read_json: Vec<JsonRestaurant> =
-        serde_json::from_reader(reader).expect("Can't parse json");
-
+    let read_json: Vec<JsonRestaurant> = serde_json::from_reader(reader).expect("Can't parse json");
```

Comment:
Why did you format the imports to be shorter, but on this line, you reformatted the line to be longer?

Reply:
ikke noget jeg gør, det er code der laver formatting hver gang jeg gemmer, så der kommer mange ændringer på grund af det